livekit_models.proto

```diff
@@ -142,6 +142,7 @@ message TrackInfo {
   // true if RED (Redundant Encoding) is disabled for audio
   bool disable_red = 15;
   Encryption.Type encryption = 16;
+  string stream = 17;
```

Comment:
would this be randomly generated? if so should it be called stream_id ?

Reply:
Currently, it is more like a stream name that generated by sfu automatically from the source:

```
func StreamFromTrackSource(source livekit.TrackSource) string {
	// group camera/mic, screenshare/audio together
	switch source {
	case livekit.TrackSource_SCREEN_SHARE:
		return "screen"
	case livekit.TrackSource_SCREEN_SHARE_AUDIO:
		return "screen"
	case livekit.TrackSource_CAMERA:
		return "camera"
	case livekit.TrackSource_MICROPHONE:
		return "camera"
	}
	return "unknown"
}
```